src/database.tf

```diff
@@ -91,3 +101,41 @@ module "postgresql" {
 #   charset             = "UTF8"
 #   collation           = "English_United States.1252"
 # }
+
+
+module "postgres_flexible_server" {
+
+  count = contains(["d"], var.env_short) ? 1 : 0
```

Comment:
Better user a boolean variable flexible_server_enabled

Reply:
Thank you for your review and suggestions. I'll fix